UnderSea.API/Startup.cs

```diff
@@ -66,49 +82,93 @@ public void ConfigureServices(IServiceCollection services)
             });
             services.AddAuthorization();
 
-            services.AddControllers();           
-            
+            services.AddControllers(options =>
+                options.Filters.Add(new HttpResponseExceptionFilter()));
+
             services.AddSwaggerGen(c =>
             {
                 c.SwaggerDoc("v1", new OpenApiInfo { Title = "UnderSea", Version = "v1" });
+
+                c.AddSecurityDefinition("basicAuth", new OpenApiSecurityScheme
+                {
+                    Type = SecuritySchemeType.ApiKey,
+                    Scheme = "Basic",
+                    Name = "Authorization",
+                    Description = "Authorization header �rt�ke. P�ld�ul: \"bearer f3g5h6345jfh634jh645\"",
+                    In = ParameterLocation.Header,
+                });
+
+                c.AddSecurityRequirement(new OpenApiSecurityRequirement
+                    {
+                        {
+                            new OpenApiSecurityScheme
+                            {
+                                Reference = new OpenApiReference {
+                                    Type = ReferenceType.SecurityScheme,
+                                    Id = "basicAuth" }
+                            }, new List<string>() }
+                    });
             });
-                  
+
+            services.AddHangfire(config =>
+                                config.SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
+                                .UseSimpleAssemblyNameTypeSerializer()
+                                .UseDefaultTypeSerializer()
+                                .UseMemoryStorage()
```

Comment:
Itt db-ben kellene tárolni a Hangfire-os adatokat is, mivel egy új telepítés alatt elvesznek az adatok, illetve limitált a memória a szerveren.